base/fitness_predictors.py

```diff
@@ -39,7 +40,12 @@ def setup_predictor(self, tree, pred, timepoint):
         if pred == 'ne_star':
             self.calc_nonepitope_star_distance(tree)
         if pred == 'tol':
-            self.calc_tolerance(tree, attr = 'tol')
+            self.calc_tolerance(tree, preferences_file='metadata/2017-12-07-H3N2-preferences-rescaled.csv', attr = 'tol')
```

Comment:
The file `2017-12-07-H3N2-preferences-rescaled.csv` has not been committed. Can you make a note of this?